lib/tasks/trends.rake

```diff
@@ -13,7 +13,7 @@ namespace :trends do
     current_record = 0
 
     trends.each do |trend_id|
-      puts "Working on trend_id #{trend_id}, #{(current_record += 1)} of #{trends_count}"
+      puts "Working on trend_id #{trend_id}, #{(current_record += 1)} of #{trends_count}" unless Rails.env.test?
```

Comment:
I'm for this, it's annoying, and should be fixed, if we're at it, let's do this as well